frontend/src/RecordingPageComp/RecordingComp.jsx

```diff
@@ -1,20 +1,27 @@
 import { AudioRecorder, useAudioRecorder } from "react-audio-voice-recorder";
-import { useContext } from "react";
+import { useContext, useRef } from "react";
 import { UserContext } from "../UserContext.js";
+import { PitchDetector } from "pitchy";
 
 function RecordingComp() {
   const { user } = useContext(UserContext);
 
   const recorderControls = useAudioRecorder();
   const reader = new FileReader();
+  const audioContextRef = useRef(new AudioContext()); // Use AudioContext for Web-Audio functions
 
   const addAudioElement = (blob) => {
-    reader.readAsDataURL(blob);
+    const MAX_SIZE = 0.25 * 1024 * 1024; // .25mb max size
```

Comment:
Curious how you came up with this size limit. Is this relate mathematically to the length (or number of seconds) of the audio recording?